### PR TITLE
add parse protobuf parameters by rust C++ ffi interface

### PR DESCRIPTION
**Pull request by fqliao**

### CODE REVIEW

File: libprecompiled/extension/wedpr-generated/asset_hiding/ffi_hidden_asset.h [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#include <cstdarg>
+#include <cstdint>
+#include <cstdlib>
+#include <new>
+
+extern "C" {
+
+char* get_credit_storage_by_fulfill_argument(char* fulfill_argument_pb);
+
```
Comment (qyan-dev) [inline libprecompiled/extension/wedpr-generated/asset_hiding/ffi_hidden_asset.h]: Remove emtpy file here.
Add a TODO to add comment later for those public FFI API

Comment (fqliao) [reply libprecompiled/extension/wedpr-generated/asset_hiding/ffi_hidden_asset.h]: The file is generated.

File: libprecompiled/extension/WedprPrecompiled.cpp [hunk 2]
```diff
@@ -84,14 +84,15 @@ bytes WedprPrecompiled::call(
...
+        if (verify_issued_credit(issue_argument_pb_char) == FAILURE)
         {
             logError(PRECOMPILED_NAME, "verify_issued_credit", VERFIY_FAILED);
             throwException(VERFIY_FAILED);
         }
 
-        std::string current_credit;
-        std::string credit_storage;
+        std::string current_credit = get_current_credit_by_issue_argument(issue_argument_pb_char);
+        std::string credit_storage = get_storage_credit_by_issue_argument(issue_argument_pb_char);
```
[the remark this answers is not in the document]
Comment (fqliao) [reply libprecompiled/extension/WedprPrecompiled.cpp]: formated by clang-format.

File: libprecompiled/extension/WedprPrecompiled.cpp [hunk 3]
```diff
@@ -84,14 +84,15 @@ bytes WedprPrecompiled::call(
...
+        char* issue_argument_pb_char = string_to_char(issue_argument_pb);
+        if (verify_issued_credit(issue_argument_pb_char) == FAILURE)
         {
             logError(PRECOMPILED_NAME, "verify_issued_credit", VERFIY_FAILED);
             throwException(VERFIY_FAILED);
         }
 
-        std::string current_credit;
-        std::string credit_storage;
+        std::string current_credit = get_current_credit_by_issue_argument(issue_argument_pb_char);
```
[the remark this answers is not in the document]
Comment (fqliao) [reply libprecompiled/extension/WedprPrecompiled.cpp]: formated by clang-format.

**Pull request merged**